index.html

```diff
@@ -696,7 +696,7 @@ <h1 class="component_heading">Font family</h1>
                 <div class="component_code">
                     <iframe
                         src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=one-dark&wt=none&l=auto&width=1080&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%27https%253A%252F%252Ffonts.googleapis.com%252Fcss2%253Ffamily%253DCaveat%2526family%253DCormorant%252BGaramond%253Awght%2540400%253B500%2526family%253DEB%252BGaramond%2526family%253DIndie%252BFlower%2526family%253DKalam%253Awght%2540300%2526family%253DLibre%252BBaskerville%2526family%253DMarck%252BScript%2526family%253DMontagu%252BSlab%253Awght%2540300%2526family%253DMontserrat%253Aital%252Cwght%25400%252C100%253B0%252C300%253B0%252C400%253B1%252C500%2526family%253DNoto%252BSerif%2526family%253DPhilosopher%253Awght%2540700%2526family%253DQuintessential%2526family%253DSource%252BSerif%252BPro%253Aital%252Cwght%25401%252C300%2526family%253DYuji%252BMai%2526display%253Dswap%27%29%253B"
-                        style="width: 1024px; height: 264px; border:0; transform: scale(1); overflow:hidden;"
+                        style="width: 1124px; height: 264px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
Would suggest using rem here for the carbon iframe for responsiveness

Reply:
noted. will change